src/common/prometheus/prometheus.service.ts

```diff
@@ -299,15 +300,21 @@ export class PrometheusService implements OnApplicationBootstrap {
     labelNames: ['nos_name'],
   });
 
+  public otherSyncParticipationAvgPercent = this.getOrCreateMetric('Gauge', {
+    name: METRIC_OTHER_SYNC_PARTICIPATION_AVG_PERCENT,
+    help: 'Other sync committee validators participation avg percent',
+    labelNames: [],
+  });
+
   public chainSyncParticipationAvgPercent = this.getOrCreateMetric('Gauge', {
     name: METRIC_CHAIN_SYNC_PARTICIPATION_AVG_PERCENT,
-    help: 'Other sync committee validators participation avg percent',
+    help: 'Chain sync committee validators participation avg percent',
```

Comment:
Does **chain** stand of overall network?

Reply:
Exactly